waku/v2/node/wakunode2.nim

```diff
@@ -747,8 +754,11 @@ when isMainModule:
     waitFor connectToNodes(node, conf.staticnodes)
 
   # NOTE Must be mounted after relay
-  if conf.lightpush:
+  if (conf.lightpushnode != "") or (conf.lightpush):
```

Comment:
Is it possible that `lightpushnode`   is passes but not `lightpush`? I think checking `lightpush` would be enough, also to disallow wrong set of inputs (i.e.., light push is not set but light push node is passed).

Reply:
It is possible, yes. Absolutely agree with you that the two settings should probably be clarified. For now I've just kept this consistent with how `filter` is also done elsewhere (setting either `filternode`, `filter` or both).

In future, though, a "client" for either filter/lightpush may want to set only the server-side as node without providing server capabilities itself (i.e. not really mounting the protocol in the same way). Since we don't have this clear separation in config yet, I'm allowing either setting to be used.